pkg/builtinchecks/yamls/minimum-replicas.yaml

```diff
@@ -0,0 +1,10 @@
+name: "required-minimum-replicas"
```

Comment:
This name should have the number three in it, since that's what it checks for. (The recommendation is not a global standard, but one that we are making, so it seems better to be explicit.)

Reply:
Fixed